packages/scene-composer/src/components/panels/SceneHierarchyPanel/components/SceneHierarchyTree/SceneHierarchyTreeItem.tsx

```diff
@@ -85,7 +85,7 @@ const SceneHierarchyTreeItem: FC<SceneHierarchyTreeItemProps> = ({
       expandable={childRefs.length > 0}
       selected={selected === key}
       selectionMode={selectionMode}
-      onSelected={isViewing() ? onActivated : onToggle}
+      onSelected={onToggle}
```

Comment:
is this the new expected behavior to not move camera on select in viewing mode?

Reply:
As I understand it, the camera should only move to target on a double-click. Single-click should update what is populated in the inspect panel so that the user can maintain view on the item they want. Ex: Double-click to target the fork lift, then single-click the directional light. The user can then modify the light intensity, color, etc... to see the effect it has on the scene.

Reply:
this is more for viewing mode, not editing mode. for editing mode, we don't move camera with single click, but the current behavior for viewing mode is that we will move camera to the object when it's single clicked

Reply:
Will revert this change. If someone wants it to behave differently we'll get some documentation for it.